sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -1166,6 +1168,8 @@ public static Sink sink(Schema schema) {
 
       abstract Builder setRowGroupSize(int rowGroupSize);
 
+      abstract Builder setAvroDataModel(Class<? extends GenericData> modelClass);
```

Comment:
The similar question - can we just pass `GenericData` here because of `withAvroDataModel(GenericData)`?